### ISSUE TITLE
vocab.json for WikiText-103

### ISSUE THREAD

**Issue by wenchenvincent** [graphcore-research/unit-scaling-demo#1]
Hello.
The downloaded WikiText-103 only has the following three files:
wiki.test.raw  wiki.train.raw  wiki.valid.raw

It seems that the run_experiment.py requires the dictionary file vocab.json. Is there a script for generating vocab.json?

**Comment by DouglasOrr** [comment]
Hi! Thanks for asking; my apologies—I should have uploaded the conversion scripts already.

The dataset can be found [here](https://graphcore-research-public.s3.eu-west-1.amazonaws.com/2023-moe-notebook/wikitext103.zip).

For reference, I've also uploaded the script we used to convert the original dataset [here](https://github.com/graphcore-research/unit-scaling-demo/blob/main/Dataset.ipynb).

(If it's of any interest, we have just recently released an early version of a more reusable library of [unit scaling components for PyTorch](https://github.com/graphcore-research/unit-scaling).)
